js/google_ima.js

```diff
@@ -1150,8 +1178,6 @@ require("../html5-common/js/utils/utils.js");
        */
       var _IMA_SDK_destroyAdsManager = privateMember(function()
       {
-        this.currentIMAAd = null;
-        this.currentNonLinearIMAAd = null;
```

Comment:
Yep, I've decoupled the IMAAd objects from the destruction of the ads manager. These two will be nulled if necessary at points after destroyAdsManager is called in the code.